BenchPress/Helpers/Azure/KeyVault.psm1

```diff
@@ -0,0 +1,234 @@
+<#
+.SYNOPSIS
+  Helper function for KeyVault
+
+.DESCRIPTION
+  Helper function for KeyVault
+
+.PARAMETER name
+  The name of the KeyVault
+
+.PARAMETER resourceGroupName
+  The name of the resource group
+
+.EXAMPLE
+  Get-KeyVault -name "kvbenchpresstest" -resourceGroupName "rgbenchpresstest"
+#>
+function Get-KeyVault {
+  [CmdletBinding()]
+  param (
+    [Parameter(Mandatory = $true)]
+    [string]$name,
+
+    [Parameter(Mandatory = $true)]
+    [string]$resourceGroupName
+  )
+
+  $resource = Get-AzKeyVault -ResourceGroupName $resourceGroupName -VaultName $name
+  return $resource
+}
+
+<#
+.SYNOPSIS
+Helper function for KeyVault
+
+.DESCRIPTION
+  Helper function for KeyVault
```

Comment:
NIT: Perhaps a more descriptive text could be used here. Something along the lines of "Name of the Keyvault that will be tested" or something.